run/storage-streaming/chunked-http.js

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START functions_tips_chunked_download]
+// [START cloudrun_tips_chunked_download]
+// [START functions_tips_chunked_upload]
+// [START cloudrun_tips_chunked_upload]
+const uuidv4 = require('uuid').v4;
+const Busboy = require('busboy');
+
+const {Storage} = require('@google-cloud/storage');
+const storage = new Storage();
+
+const sourceFile = storage.bucket('cloud-devrel-public').file('data.txt');
+
+// [END functions_tips_chunked_download]
+// [END cloudrun_tips_chunked_download]
+// TODO(developer): set this to a bucket you own
+const TARGET_BUCKET = process.env.FUNCTIONS_BUCKET;
+// [END functions_tips_chunked_upload]
+// [END cloudrun_tips_chunked_upload]
+// [START functions_tips_chunked_download]
+// [START cloudrun_tips_chunked_download]
+
+// This function downloads a file from Google Cloud Storage with HTTP chunking.
+// This lets a function send files that are bigger than the instance's available
+// memory.
+exports.chunkedDownload = async (req, res) => {
+  const readStream = sourceFile.createReadStream();
+
+  // Download file to the end-user's device
+  // TODO(developer): remove this to send files (e.g. images) directly to users
+  res.attachment('data.txt');
+
+  readStream.pipe(res);
```

Comment:
nit: add comment similar to: "pipes all of the data from the readable into the response"

Reply:
Done.